torch/csrc/api/src/data/htk/htk_parser.cpp

```diff
@@ -0,0 +1,117 @@
+#include "htk_parser.hpp"
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+
+using boost::property_tree::ptree;
+using boost::property_tree::read_json;
```

Comment:
We need to check this with pytorch code. I am not sure if they like to take boost.

Reply:
I think we discussed about this before. You said this reader can be a plug-in like CNTK's case, and it can consume pyTorch like a submodule?